Shared/Samples/Augment reality to fly over scene/AugmentRealityToFlyOverSceneView.swift

```diff
@@ -59,6 +60,8 @@ private extension URL {
     }
 }
 
+#if !targetEnvironment(macCatalyst)
 #Preview {
     AugmentRealityToFlyOverSceneView()
 }
+#endif
```

Comment:
I wonder if we should remove the preview - even if the scene view shows, it doesn't convey the concept of this sample when previewed in a simulator 🤔

Reply:
Seems like the best thing to do is remove AR previews.